nibabel/freesurfer/mghformat.py

```diff
@@ -459,9 +459,13 @@ class MGHImage(SpatialImage):
     """ Class for MGH format image
     """
     header_class = MGHHeader
-    files_types = (('image', '.mgh'),)
+    files_types = (('image', '.mgh'),
+                   ('image', '.mgz'))
```

Comment:
If we go the `alternate_exts` route, would `Spm99AnalyzeImage`'s use of the `mat` extension fit in here?

```
    files_types = (('image', '.img'),
                   ('header', '.hdr'),
                   ('mat', '.mat'))
```

I'm not familiar with this, but I don't think this fits the definition for `files_types` that @matthew-brett outlined above. It may fit the `alternate_exts` idea?